### PR TITLE
Add code debugging document under windows Intellj IDEA

### PR DESCRIPTION
**Pull request by hf200012**
Add documentation for debugging and running fe in the Windows Intellj IDEA development environment

### CODE REVIEW

File: docs/zh-CN/developer-guide/idea-debug.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# apache doris windows下fe开发环境搭建
```
Comment (EmmyMiao87) [inline docs/zh-CN/developer-guide/idea-debug.md:1]: Add license and title like other document

**Comment by EmmyMiao87** [comment]
Please add english document also. You can use google or baidu translate directly.

File: docs/zh-CN/developer-guide/idea-debug.md (hunk 1 above)
Comment (morningman) [reply docs/zh-CN/developer-guide/idea-debug.md:1]: Better rename title to
`使用 IntelliJ IDEA 搭建 FE 开发环境`

File: docs/zh-CN/developer-guide/fe-idea-dev.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+在linux下，进入到源码目录，执行下面的命令：
+
+```
+$ sh build.sh --clean --fe
+```
+
+然后将 gensrc目录打包，拿出来，如下图
+
+![image-20200624124200704](../../.vuepress/public/images/DEBUG1.png)
```
Comment (morningman) [inline docs/zh-CN/developer-guide/fe-idea-dev.md:43]:
```suggestion
![image-20200624124200704](/images/DEBUG1.png)
```
这样就可以了

**Review by morningman — APPROVED** [review]
LGTM
There are still some problem about this doc. But for the first issue, it is OK.
I will modify it in another PR later.

**Pull request merged**